sklearn/model_selection/tests/test_search.py

```diff
@@ -1305,3 +1307,12 @@ def _pop_time_keys(cv_results):
                                   per_param_scores[1])
         assert_array_almost_equal(per_param_scores[2],
                                   per_param_scores[3])
+
+
+def test_inverse_transform():
+    clf = MockClassifier()
+    grid_search = GridSearchCV(clf, {'foo_param': [1, 2, 3]}, verbose=3)
+
+    grid_search.fit(X, y)
+    inverse_transform = grid_search.inverse_transform(X)
+    assert_array_equal(X, inverse_transform)
```

Comment:
Rather than assuming what inverse_transform is I think it would be better to check that `inverse_transform(transform(X)) == X`.

You could implement `transform` to be `X + self.foo_param` and `inverse_transform` to be `X - self.foo_param` for example.